tsconfig.json

```diff
@@ -16,14 +16,21 @@
     ],
     "strict": true,
     "strictNullChecks": true,
-    "lib": ["es6", "es2015", "dom"]
+    "lib": [
+      "es6",
+      "es2015",
```

Comment:
**TODO** I believe we can change this to 'ES2022' and 'dom'

Reply:
Resolved in 020928374c41ed2bfe0c8fa785eb1ed410457ccf